changelog/is_input_range_element.dd

```diff
@@ -0,0 +1,7 @@
+isInputRange now takes an optional element type.
+
+isInputRange now takes an optional 2nd template parameter that
+defaults to void. If not void, it only evaluates to true if the
+range's element type is convertible to passed-in parameter. For
+instance, `isInputRange!(int[], int)` is true, but
+`isInputRange!(int[], string)` is false.
```

Comment:
The changelog entry needs to be updated to match the new implementation. It talks about being convertible in general rather than saying anything about constness, which is the only type of conversion it now allows.

Also, if I'm being pedantic, `isInputRange` "now takes an option 2nd template argument" - or "now has an option 2nd template parameter", since the parameter is what the template or function has in its signature, whereas the argument is what you pass to it. Similarly, you wouldn't have a "passed-in parameter" but a "passed-in argument." However, given how often folks don't bother to use those terms correctly, I don't know many people will even notice.